content/docs/en/elements/components/search-bar.md

```diff
@@ -1,5 +1,28 @@
 ---
 title: SearchBar
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_button_.button
```

Comment:
should be: https://docs.nativescript.org/api-reference/classes/_ui_search_bar_.searchbar

Reply:
At this point you'll notice that I was using the button template and changing things... haha